src/features/student/selfKnowledge/components/interactions/formFields/CategoryElementDescriptionTextareaFormField/CategoryElementDescriptionTextareaFormField.test.ts

```diff
@@ -0,0 +1,135 @@
+import type { AddSelfKnowledgeCategoryElementForm } from '@/features/student/selfKnowledge/types/forms.types'
+import CategoryElementDescriptionTextareaFormField
+  from '@/features/student/selfKnowledge/components/interactions/formFields/CategoryElementDescriptionTextareaFormField/CategoryElementDescriptionTextareaFormField.vue'
+import { CategoryElementDescriptionTextareaStub } from '@/features/student/selfKnowledge/components/interactions/inputs/CategoryElementDescriptionTextarea/CategoryElementDescriptionTextarea.stub'
+import { BddTest } from '@avenirs-esr/avenirs-dsav/test-utils'
+import { useForm } from '@tanstack/vue-form'
+import { mount, type VueWrapper } from '@vue/test-utils'
+import { beforeEach, expect, vi } from 'vitest'
+
+const TestWrapper = {
+  components: {
+    CategoryElementDescriptionTextareaFormField
+  },
+  setup () {
+    const form = useForm({
+      defaultValues: { title: '', description: '', rating: null },
+      validators: {
+        onSubmit () {
+          return { fields: { description: undefined } }
+        }
+      }
+    }) as unknown as AddSelfKnowledgeCategoryElementForm
+
+    return { form }
+  },
+  template: `
+    <form @submit.prevent="form.handleSubmit">
+      <CategoryElementDescriptionTextareaFormField :form="form" />
+    </form>
+  `
+}
+
+BddTest().given('a self knowledge category element description textarea form field component', () => {
+  let wrapper: VueWrapper
+
+  const stubs = {
+    CategoryElementDescriptionTextarea: CategoryElementDescriptionTextareaStub
+  }
+
+  beforeEach(() => {
+    vi.clearAllMocks()
+    wrapper = mount(TestWrapper, {
+      global: { stubs }
+    })
+  })
+
+  BddTest().when('the component is mounted', () => {
```

Comment:
Pour être logique il faudrait faire comme ça

```suggestion
  BddTest().when('the component is mounted', () => {
    beforeEach(() => {
      vi.clearAllMocks()
      wrapper = mount(TestWrapper, {
        global: { stubs }
      })
    })
```